packages/lib-classifier/src/store/markings/PointStore.js

```diff
@@ -0,0 +1,84 @@
+import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
+import { autorun } from 'mobx'
+import { filter } from 'rxjs/operators'
+
+import Mark from './Mark'
+
+const Point = types
+  .model('PointStore', {
```

Comment:
Same comment about naming here. I'm not sure if this model represents a point or a store to hold points.